### ISSUE TITLE
ORA-04030: out of process memory when trying to allocate nn bytes

### ISSUE THREAD

**Issue by fddc** [mbleron/ExcelGen#46]
When I use the putCell procedure to generate xlsx, If there are too many rows(half a million e.g), this error will occur

**Comment by mbleron** [comment]
You're exhausting the PGA by putting that many cells in memory.
What is the use case? Can't you do the same using a table approach?

**Comment by fddc** [comment]
Yes, I can do the same using a table approach, but i need to write same query SQL twice.
If I could use putRow in the loop logic, that would be more elegant.
Maybe we can use CLOB global variable to store putCell temporary datas?

Thanks for your project, this framework is very helpful.

**Comment by mbleron** [comment]
We may consider using a temporary table for this scenario, GTT or PTT, depending on the db version.
Not at the top of my list at the moment though.
